Tasks/HelmDeployV0/src/utils.ts

```diff
@@ -52,4 +52,10 @@ function ensureDirExists(dirPath : string) : void {
     if (!fs.existsSync(dirPath)) {
         fs.mkdirSync(dirPath);
     }
+}
+
+export function getHelmPathForACR() {
+    const chartName = tl.getInput("chartName", true);
```

Comment:
What if chartType is chartPath?

Reply:
We will not have the input field visible for this command.